front/src/components/mmt-waypoint-card/mmt-waypoint-card.component.js

```diff
@@ -0,0 +1,38 @@
+export default {
+  data () {
+    return {
+      iconMap: [
+        'mdi-airplane',
+        'mdi-train',
+        'mdi-bus',
+        'mdi-car',
+        'mdi-bicycle'
+      ]
+    };
+  },
+  props: {
+    lastCard: {
+      type: Boolean,
+      default: false
+    },
+    firstCard: {
+      type: Boolean,
+      default: false
+    },
```

Comment:
it is not what i expect from card component. card should render only card, no other things and it don't need to know its order
it seems just it is not only card - so, separate non card part to other component

Reply:
But first card should not contain arrival date, last card - departure date